src/tools/rustfmt/src/expr.rs

```diff
@@ -243,7 +243,8 @@ pub(crate) fn format_expr(
         ast::ExprKind::Try(..)
         | ast::ExprKind::Field(..)
         | ast::ExprKind::MethodCall(..)
-        | ast::ExprKind::Await(_, _) => rewrite_chain(expr, context, shape),
+        | ast::ExprKind::Await(_, _)
+        | ast::ExprKind::Yield(_, ast::YieldKind::Postfix) => rewrite_chain(expr, context, shape),
```

Comment:
Does calling `rewrite_chain` just work? I would have though that we'd need to make some changes to `src/tools/rustfmt/src/chain.rs` and add a new `ChainItemKind` variant for `Yield`.

Reply:
You're correct, there are some more changes needed. I'm working on them on my machine but haven't pushed them up yet.

Reply:
I've pushed the rustfmt changes now.

Reply:
@eholk thanks. I looked them over and they seem fairly straightforward! I think we're good to go on the rustfmt changes.